slam/src/slam/mrclam/RunMRCLAMDataset.py

```diff
@@ -0,0 +1,156 @@
+import warnings
+import os
+warnings.filterwarnings("ignore")
+warnings.filterwarnings("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore", category=UserWarning)
+import pickle
+import pandas as pd
+from slam.mrclam.MRCLAMDataloader import *
+from slam.fast_slam2.FastSLAM2 import FastSLAM2
+import numpy as np
+from slam.Util import wrap_to_pi, plot_df
+from slam.fast_slam2.Models import Meas, FastSLAM2Parameters, LandmarkConstants
+from slam.robot_physics.RobotPhysics2D import RobotPhysics2D
+
+class RunMRCLAMDataset():
```

Comment:
this file is rather straightforward but esp for newcomers getting into the code base, a comment or two would help